src/pages/Build/DataPage.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<www-page>
+		<developer-secondary-menu slot="secondary" />
+		<div class="row page-content">
+			<div class="small-12 columns">
+				<h1>Data Snapshots</h1>
+				<div class="callout secondary">
+					<strong class="show-for-medium-up">Important Note:</strong>
```

Comment:
Can we add some margin

Reply:
added .5rem to the bottom. Thanks for 👀 !